ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -488,3 +488,11 @@ def __iter__(self):
 
     def __mod__(self, value, /):
         return np_frontend.mod(self, value, out=self)
+    
+    def ptp(self, *, axis=None, out=None, keepdims=False):
+        if isinstance(self.ivy_array, tuple):
+            xmax = ivy.max(self.ivy_array)
```

Comment:
Also, keep in mind you should be using all the arguments to get the correct results. You are currently forgetting about `axis` and `keepdims`.